ci/license_exceptions.txt

```diff
@@ -15,3 +15,7 @@ certifi
 # ordereddict is MIT according to https://pypi.python.org/pypi/ordereddict but
 # apparently the metadata is lacking (it shows up as UNKNOWN)
 ordereddict
+
+# pyzmq/zmq is BSD. Metadata missing.
+pyzmq
```

Comment:
These aren't in requirements.txt? Remove them?

Reply:
done

Reply:
This is still here. Did you mean to remove it?

Reply:
done